sdk/openai/openai/samples/v1-beta/javascript/completionsRest.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
This is sample is not publishable at the moment because it needs a special package.json file and dev-tool is not aware of that requirement yet. Could we unpublish it by deleting it from the samples-dev folder before publishing? I am also open to remove them from samples-dev to somewhere else for now, e.g. samples-dev-module.

Reply:
Hmm, I'll go ahead and move it to a new `samples-dev-module` folder then.